configure

```diff
@@ -2980,6 +2992,8 @@ case $_backend in
 		LIBS="$LIBS -lpng"
 		LIBS="$LIBS -Wl,-Map,mapfile.txt"
 		;;
+	raspberrypi)
+		;;
 	samsungtv)
```

Comment:
This is where you should set DEFINES, INCLUDES and LIBS (again: take a look at the surrounding code). Also, make sure to imitate what the surrouding code does to add to these vars: It adds one entry per line. E.g. it does
```
		LDFLAGS="$LDFLAGS -shared"
		LDFLAGS="$LDFLAGS -fpic"
``
This way, when a flag is added or removed, it shows up nicely in diffs.